AmahiAnywhere/AmahiAnywhere/Presentation/Settings/SettingsViewController.swift

```diff
@@ -35,7 +35,7 @@ class SettingsViewController: BaseUITableViewController, MFMailComposeViewContro
         self.present(sendMailAlert, animated: true)
     }
     
-    func mailComposeController(_ controller:MFMailComposeViewController,
+    private func mailComposeController(_ controller:MFMailComposeViewController,
```

Comment:
I'm sorry, I missed this part while reviewing. Please remove private from here otherwise this function is not executed. @codedentwickler   

To check this:
Open settings -> Tell a friend (or) Feedback -> Send (or) Cancel

Reply:
Thanks for pointing that out. I'll fix it in a PR.